packages/react/src/components/Table/TableBody/SkeletonRow.test.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import { settings } from '../../../constants/Settings';
+
+import SkeletonRow from './SkeletonRow';
+
+const { prefix } = settings;
+
+const skeletonRowProps = () => ({
+  id: 'skeletonRow01',
+  tableId: 'tableId',
+  testId: 'skeleton-row-test',
+  rowVisibilityRef: null,
+  hasRowActions: false,
+  hasRowExpansion: false,
+  hasRowNesting: false,
+  hasRowSelection: false,
+  showExpanderColumn: false,
+  columns: [
+    {
+      id: 'string',
+      name: 'String',
+      isSortable: true,
+    },
+    {
+      id: 'select',
+      name: 'Select',
+      isSortable: true,
+    },
+    {
+      id: 'number',
+      name: 'Number',
+      isSortable: true,
+    },
+  ],
+});
+
+describe('SkeletonRow', () => {
+  it('creates standard data skeleton row', () => {
+    const { container } = render(<SkeletonRow {...skeletonRowProps()} />, {
+      container: document.body.appendChild(document.createElement('tbody')),
+    });
+    expect(container).toBeDefined();
+    expect(container.rows[0].cells.length).toBe(3);
```

Comment:
just a note, you could also do `expect(container.querySelectorAll('td')).toHaveLength(3)`